egs/wsj/s5/steps/dict/learn_lexicon.sh

```diff
@@ -168,7 +168,8 @@ if [ $stage -le 0 ]; then
     # create an empty list of g2p generated prons, if it's not given.
     touch $dir/lexicon_g2p.txt
   else
-    cp $lexicon_g2p $dir/lexicon_g2p.txt 2>/dev/null
+    cat $lexicon_g2p | awk '{if (NF<2) {print "There is an empty pronunciation in lexicon_g2p.txt." \
+      > "/dev/stderr"; exit 1} print NF}' > $dir/lexicon_g2p.txt || exit 1;
```

Comment:
Also, this 'print NF' is not right.  It should be 'print $0'.  I think maybe you should test this.